amavisvt/client.py

```diff
@@ -489,6 +489,51 @@ def is_included(resource):
                     lambda r: '.' not in r.filename or r.filename.endswith('.')
         ]))
 
+    def api_call_check(self, checksums):
+       for api_key in self.config.apikeys:  
+           use_api_key = self.memcached.get(api_key)
+           if not use_api_key:
+               response = requests.post(self.config.api_url, {
```

Comment:
Please reformat these arguments to be more compliant with the python style guide.